src/components/RequestsFilters/RequestsFilters.test.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import RequestsFilters from './RequestsFilters';
+
+jest.mock('@folio/stripes/components', () => ({
+  AccordionSet: jest.fn(({ children }) => <div>{children}</div>),
+  Accordion: jest.fn(({ displayClearButton, name, children, onClearFilter }) => (
+    <div>
+      <div>{displayClearButton ? <button type="button" onClick={() => onClearFilter()} data-testid={`clear-${name}`}>Clear</button> : null}</div>
+      <div data-testid={`accordion-${name}`}>{children}</div>
+    </div>
+  )),
+  FilterAccordionHeader: jest.fn(() => <div>FilterAccordionHeader</div>),
+}));
+
+jest.mock('@folio/stripes/smart-components', () => ({
+  CheckboxFilter: jest.fn(() => <div>CheckboxFilter</div>),
+  MultiSelectionFilter: jest.fn(() => <div>MultiSelectionFilter</div>),
+}));
+
+const props = {
+  activeFilters: {
+    requestStatus: ['Open'],
+    requestType: ['Hold'],
+    pickupServicePoints: ['1'],
+    tags: ['Urgent'],
+  },
+  onChange: jest.fn(),
+  onClear: jest.fn(),
+  resources: {
+    servicePoints: { records: [{ id: '1', name: 'Service Point 1' }] },
+    tags: { records: [{ label: 'Urgent' }] },
+  },
+  titleLevelRequestsFeatureEnabled: false,
+};
+
+describe('RequestsFilters', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it('renders the request type filter', () => {
+    const { getByTestId } = render(<RequestsFilters {...props} />);
+    const requestTypeButton = getByTestId('clear-requestType');
+    expect(requestTypeButton).toBeInTheDocument();
+    userEvent.click(requestTypeButton);
+  });
+  it('renders the request status filter', () => {
+    const { getByTestId } = render(<RequestsFilters {...props} />);
+    const requestStatusButton = getByTestId('clear-requestStatus');
+    expect(requestStatusButton).toBeInTheDocument();
+    userEvent.click(requestStatusButton);
```

Comment:
What is the reason to trigger `click` event here?
Is it necessary only for increasing code coverage?